### DISCUSSION TITLE
Best way to configure minisearch to not split at - Sign (Hyphen, Minus)

### DISCUSSION THREAD

**Discussion by gregorwolf** [vuejs/vitepress#2569] [Q&A] [answered]
Hello VitePress Community,

as described in [How to search for items with an embedded hyphen? #108](https://github.com/lucaong/minisearch/issues/108) minisearch does split by default at Hyphen/Minus Sign. It is possible to influence this behaviour by defining a [custom split variable](https://github.com/lucaong/minisearch/issues/108#issuecomment-963311944). Is there a place where this could be done in VitePress?

Best Regards
Gregor

**Comment by brc-dd** [comment, marked as answer]
Not yet, feel free to create a [feature request](https://github.com/vuejs/vitepress/issues/new/choose) or a PR.

**Reply by gregorwolf** [reply]
Done: [Allow configuration of minisearch to not split at - Sign (Hyphen, Minus) #2570](https://github.com/vuejs/vitepress/issues/2570). Thank you for the guidance.
